gui/gui.go

```diff
@@ -502,9 +493,13 @@ Buffer Size: %d lines
 		}
 	}
 
-	close(stop) // Tell waker to end.
+	gui.logger.Debug("Stopping render...")
+
+	close(stop)      // Tell waker to end...
+	<-gui.wakerEnded // ...and wait it to end
```

Comment:
It would be cleaner to use a `sync.WaitGroup` for this. The WaitGroup could also be used for the logger.Sync goroutine. Also, the logger.Sync goroutine should be stopped using the same channel used to stop the waker goroutine.

Reply:
Agreed, will change this to use `sync.WaitGroup`

Reply:
done in commit https://github.com/liamg/aminal/pull/262/commits/d65f22c819284b187b89a22282b74dc0fb22fcc5